### DISCUSSION TITLE
ConfuserEx 1.6 Release

### DISCUSSION THREAD

**Discussion by darkroastcreative** [mkaring/ConfuserEx#443] [General]
I'm reaching up at follow up on an issue I submitted earlier this year, #348. Specifically, I am interested in knowing if there is an updated timeline for the release of ConfuserEx 1.6. This release includes a fix which my team is waiting to be released in a Production build and we are hoping we can adopt 1.6 in our product development workflows in early 2022.

I appreciate any insights you may be able to provide!

**Comment by mkaring** [comment]
I'm going to prepare the release candidate version of ConfuserEx 1.6 within the next couple of days. If nothing critical pops up, the release version will be available in mid January 2022.

**Reply by darkroastcreative** [reply]
Thank you, Martin!

**Reply by mkaring** [reply]
The release candidate is now tagged and build.

**Reply by mkaring** [reply]
@darkroastcreative: The release is now available.

**Reply by darkroastcreative** [reply]
Thank you so much, @mkaring! My team is starting the process of adopting the new release this week and will be sure to share any feedback. 😄
